src/client/functions.ts

```diff
@@ -74,12 +74,40 @@ export const getPartyColor = (party: string) =>
 export const getMajorityRaceDisplay = (feature: DistrictGeoJSON) =>
   feature.properties.majorityRace && capitalizeFirstLetter(feature.properties.majorityRace);
 
-export function formatPvi(party?: Party, value?: number): string {
-  return value !== undefined && party !== undefined
-    ? `${party.label}+${Math.abs(value).toLocaleString(undefined, {
-        maximumFractionDigits: 0
-      })}`
-    : "N/A";
+/* Creates array of party-labelled pvi bucket counts as strings */
+export function formatPviByDistrict(
+  pviBuckets: readonly (PviBucket | undefined)[] | undefined
+): readonly string[] | undefined {
+  const partyLabels = ["R", "E (Even)", "D"];
+  // Count by partyLabels
+  const bucketCounts = pviBuckets?.reduce(
+    // eslint-disable-next-line
+    (allBuckets: { [key: string]: number } | undefined, bucket: PviBucket | undefined) => {
```

Comment:
This will clean up the lint:
```suggestion
    (allBuckets: { readonly [key: string]: number } | undefined, bucket: PviBucket | undefined) => {
```